modules/core/src/controllers/transition-manager.js

```diff
@@ -13,7 +13,8 @@ const DEFAULT_PROPS = {
   transitionInterruption: TRANSITION_EVENTS.BREAK,
   onTransitionStart: noop,
   onTransitionInterrupt: noop,
-  onTransitionEnd: noop
+  onTransitionEnd: noop,
+  position: [0, 0, 0]
```

Comment:
We shouldn't add props that are not related to transition to this object.

Reply:
Done